src/pages/Browse/index.spec.tsx

```diff
@@ -480,8 +480,11 @@ describe(__filename, () => {
     expect(dispatch).not.toHaveBeenCalled();
   });
 
-  it('dispatches viewVersionFile on update if the query string contains a `path` that is different than `version.selectedPath`', () => {
-    const { renderAndUpdate, version } = setUpVersionFileUpdate({
+  // This could happen when a keyboard navigation updates the selected path. We
+  // do not want to update the selected path again, so we apply this logic to
+  // the first render (mount) only.
```

Comment:
Oh, you do have a comment 😄 Can you move this to the `loadData()` method? That's a better place for it.